code/modules/projectiles/guns/projectile/shotgun/bojevic.dm

```diff
@@ -18,8 +18,9 @@
 	unload_sound = 'sound/weapons/guns/interact/ltrifle_magout.ogg'
 	reload_sound = 'sound/weapons/guns/interact/ltrifle_magin.ogg'
 	cocked_sound = 'sound/weapons/guns/interact/ltrifle_cock.ogg'
-	damage_multiplier = 0.8
-	recoil_buildup = 30
+	damage_multiplier = 1.2 
```

Comment:
This is full auto shotgun that can murder everything with such damage values. Please don't make it too OP, crew already struggles against serbs.

Better AP is fine, but damage should remain at 0.8.